Gemfile

```diff
@@ -45,6 +45,8 @@ group :development, :test do
   gem 'rubocop'
   gem 'dotenv-rails'
 
+  gem 'factory_girl_rails', '~> 4.4.1'
```

Comment:
I think you have to add it to development for rails generate factories instead of fixtures.